js/vod-stream-embed.js

```diff
@@ -0,0 +1,23 @@
+// Video-on-demand Stream Video
+
+var ipfs_gateway_self = 'https://ipfs.io'; // IPFS gateway of this node
+var ipfs_gateway_origin = 'https://ipfs.io'; // IPFS gateway of origin stream
+
+var stream_type = 'video/mp4'; // Type of video stream
+var stream_url_ipfs = 'https://ipfs.io/ipfs/' // File path to stream with IPFS content via HTTP server
+var stream_urls_http = []; // Optional list of URLs to stream over HTTP
```

Comment:
I'm not sure I understand the flow for how the http stream will be queued, do you imagine there being a list of mirrors + potentially feeding in one?

Reply:
For recorded ones it only supports a single one via `http` URL param